app/src/main/java/org/xbmc/kore/utils/Utils.java

```diff
@@ -250,4 +253,28 @@ public void onError(int errorCode, String description) {
             }
         }, callbackHandler);
     }
+
+    public static void setLocale(Context context, String localeName) {
+        Locale locale = getLocale(localeName);
+
+        Locale.setDefault(locale);
+
+        Resources resources = context.getResources();
+
+        Configuration configuration = resources.getConfiguration();
+        configuration.locale = locale;
+
+        resources.updateConfiguration(configuration, resources.getDisplayMetrics());
+    }
+
+    public static Locale getLocale(String localeName) {
```

Comment:
https://developer.android.com/reference/java/util/Locale.html#forLanguageTag(java.lang.String)

Reply:
Currently we do not have translations requiring variant or extensions and we only have language codes and some supplemented with country codes. As we still support API 15 I think we should stick to this and prevent as much of the API if-statements as possible.
When we finally bump the minimum supported SDK to 21 or higher we can always replace this (and test it then).

Reply:
We won't remember to do this, when/if we bump to SDK 21. It's not much trouble doing an if here and using the new versions.